services/importer/lib/importer/unp.rb

```
@@ -46,6 +46,8 @@ def run(path)
       end
 
       def without_unpacking(path)
+        raise NotAFileError if File.directory?(path)
```

Hm, not sure if I'm following.

Your line would raise an exception for any valid file, right? I want to raise an exception in the case that what is sent _is not_ a file (which could happen through the Import API).

I guess that a:

````
raise NotAFileError if !File.file?(path)
`````

could work too!